apps/omg/test/omg/dependency_conformance/signature_test.exs

```diff
@@ -34,13 +36,12 @@ defmodule OMG.DependencyConformance.SignatureTest do
   @token TestHelper.generate_entity().addr
 
   setup_all do
-    {:ok, exit_fn} = Eth.DevNode.start()
+    {:ok, exit_fn} = DevNode.start()
```

Comment:
hm, I looked at it again now, at the final result, and I still have some doubts. I think getting rid of `Eth` (or other component like `Watcher` obscures things a bit. `Eth.DevNode` was more explicit than `DevNode`; `DevNode` could be anything, it's not apparent it is an Ethereum node we're talking about. 

Same goes with `Deployer`, `DevHelper`. `Eth.Support.DevNode` would be explicit, but is lengthy.

Maybe let's try `defmodule Support.Eth.DevNode` and then `alias Support.Eth` and in-code use with `Eth.DevNode`? would this work?

Reply:
Well a few things:

> DevNode could be anything, it's not apparent it is an Ethereum node we're talking about.

What other node would we be talking about here?

> Same goes with Deployer, DevHelper. Eth.Support.DevNode would be explicit, but is lengthy.

* the code base is inconsistent with how we surface module usage.
  * Sometimes we fully alias the entire module,
  * sometimes alias just the top level
  * sometimes a portion of it
  * And other times it just doesn't alias at all. 
  * or we import it.
I'm focusing more on consistency here and assuming that we always alias the full module name. If we follow the above rule, it will be much more consistent to discover what the module is by looking at our alias declarations.
* The eventual goal is to remove all this code. As we move towards docker, the hope is we don't need to rely on these elixir code to prop up our services.

I would be open to `Support.Eth.Node` but we should still fully alias the module like so:

```elixir
alias Support.Eth.DevNode
# ...
DevNode.start()
```

Reply:
> What other node would we be talking about here?

Node in a cluster, node in a tree. I'm talking about the experience of a first-time reader here. `Eth.DevNode` is pretty clear.

> the code base is inconsistent with how we surface module usage
> ...I'm focusing more on consistency here and assuming that we always alias the full module name

I don't think this little bit of consistency is worth the decrease of readability. E.g. turning `Utxo.Position` into `Position` will be detrimental, similar conflict would become of `Watcher.DB.Repo` and `Ecto.Repo` becoming `Repo` etc.

I think `Plug.Conn`, `Ecto.Repo` are commonly used non-fully-aliased module names out there.

Maybe let's settle for consistent use of a particular module throughout code, e.g. `Eth.DevNode` stays `Eth.DevNode` and `Support.Eth.DevNode` and `DevNode` are both avoided. WHat do you think?

> I would be open to Support.Eth.Node but we should still fully alias the module

That would be good, but I'd advise against forcing the full alias, see above

Reply:
Actually, in elixir's and erlangs terms, a Node is already taken. A node is a beam VM instance.

Reply:
> Maybe let's settle for consistent use of a particular module throughout code, e.g. Eth.DevNode stays Eth.DevNode and Support.Eth.DevNode and DevNode are both avoided. WHat do you think?

> Actually, in elixir's and erlangs terms, a Node is already taken. A node is a beam VM instance.

I think perhaps this is the problem, the use of the word `Node` was problematic here. I'm going to fix the merge conflicts then propose we just call this something else all together. (Perhaps something more verbose like EthereumService)

Again, the end goal is to remove all of this and move starting Ethereum outside of elixir.